src/topp/CometAdapter.cpp

```diff
@@ -609,6 +609,9 @@ class TOPPCometAdapter :
     os << "10. Chymotrypsin           1      FWYL        P" << "\n";
     os << "11. No_cut                 1      @           @" << "\n";
     os << "12. Arg-C/P                1.     R           _" << "\n";
+    os << "13. Lys-C/P                1      K           -" << "\n";
+    os << "14. Glutamyl_endopeptidase 1      E           E" << "\n";
+    os << "15. Leukocyte_elastase     1      ALIV        -" << "\n";
```

Comment:
these need to be registered with their comet ID, otherwise this will not work.
The current implementation is also incomplete, since its not defined what happens if a user selects an enzyme which is not in this list. An on-the-fly generator would be ideal ...